cms/templates/partials/nhs_components/select_search_results_by_order.html

```diff
@@ -1,22 +1,20 @@
-{# #}
 {#TODO: add functionality to make this interactive#}
 <form action="">
   <input type="hidden" name="query" value="{{ query_string }}">
   <input type="hidden" name="content_type" value="{{content_type}}">
   <input type="hidden" name="date_from" value="{{date_from}}">
   <input type="hidden" name="date_to" value="{{date_to}}">
-  <fieldset class="nhsuk-fieldset">
-    <legend class="nhsuk-fieldset__legend nhsuk-fieldset__legend--xs">
-      <h1 class="nhsuk-fieldset__heading">Order by date</h1>
+  <fieldset class="sort_by_nhsuk-fieldset nhsuk-fieldset">
+    <legend class="sort_by nhsuk-fieldset__legend nhsuk-fieldset__legend--xs">
+      <h1 class="nhsuk-fieldset__heading">Sort by:</h1>
     </legend>
     <div class="nhsuk-form-group">
       <div class="nhsuk-row nhsuk-u-margin-botom-3">
         <select class="nhsuk-select" id="publish_date" name="order"
           onchange="this.form.submit()">
-          <option value="-latest_revision_created_at"{% if order == '-latest_revision_created_at' %} selected{% endif %}>Version (newest first)</option>
-          <option value="latest_revision_created_at"{% if order == 'latest_revision_created_at' %} selected{% endif %}>Version (oldest first)</option>
-          <option value="-first_published_at"{% if order == '-first_published_at' %} selected{% endif %}>Published (newest first)</option>
-          <option value="first_published_at"{% if order == 'first_published_at' %} selected{% endif %}>Published (oldest first)</option>
+          <option value="-latest_revision_created_at"{% if order == '-latest_revision_created_at' %} selected{% endif %}>Most relevant</option>
+          <option value="latest_revision_created_at"{% if order == 'latest_revision_created_at' %} selected{% endif %}>Newest first</option>
+          <option value="-first_published_at"{% if order == '-first_published_at' %} selected{% endif %}>Oldest first</option>
```

Comment:
I'm confused as to why the Newest and Oldest would use different metrics for age; is this deliberate? (One of the things I thought people said they needed was to be able to quickly scan which documents had recently been updated or added; do we have any particular understanding which changes that?)